app/controllers/secrets_controller.rb

```diff
@@ -66,8 +66,20 @@ def batch
       unless (secret = variable.last_secret)
```

Comment:
Existing code but this should be should fixed: We shouldn't combine assignment with an unless check.